src/Toggler.js

```diff
@@ -44,9 +44,11 @@ class Toggler extends State {
 			return content;
 		}
 
-		content = header.querySelector(this.content);
-		if (content) {
-			return content;
+		if (header.querySelector) {
```

Comment:
I'll remove the verification since we probably need to mandate passing in an element for `header`